components/mailgun/mailgun.app.js

```diff
@@ -0,0 +1,192 @@
+const axios = require("axios");
+const get = require("lodash/get");
+const moment = require("moment");
+const retry = require("async-retry");
+
+module.exports = {
+  type: "app",
+  app: "mailgun",
+  propDefinitions: {
+    domain: {
+      type: "string",
+      label: "domain",
+      description: "Domain used to send and receive email.",
+      async options(context) {
+        const q = context.query;
+        const options = [];
+        const results = await this.getAllDomains();
+        for (const domains of results) {
+          options.push(domains.name);
+        }
+        return options;
+      },
+    },
+    baseRegion: {
+      type: "string",
+      label: "Base region",
+      description:
+        'The region that Mailgun will use to send and receive email, which determines the URL to use when requesting Mailgun API. This must be the same region where "domain" was created.',
+      default: "US",
+      options: ["US", "EU"],
+    },
+    timer: {
+      label: "Polling schedule",
+      description: "Pipedream polls Reddit for events on this schedule.",
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15, // by default, run every 15 minutes.
+      },
+    },
+  },
+  methods: {
+    mailgun(apiKey, domain, host) {
+      return require("mailgun-js")({ apiKey, domain, host });
+    },
+    _apiKey() {
+      return this.$auth.api_key;
+    },
+    _apiHost() {
+      const subdomain = this.baseRegion == "EU" ? "api.eu" : "api";
+      return `${subdomain}.mailgun.net`;
+    },
+    async _makeRequest(opts) {
+      const mailgun = this.mailgun(
+        this._apiKey(),
+        opts.domain,
+        this._apiHost()
+      );
+      switch (opts.method) {
+        case "get":
```

Comment:
Can you replace all instances of lowercase `get` with `GET`, `post` with `POST`, etc.? Small style thing but even though `axios` allows for the lowercase versions, we like to use the capitalized versions to align with the HTTP spec

Reply:
Yep, this was just replaced.